Elena Krasnopolskaya/Order.js

```diff
@@ -0,0 +1,32 @@
+function Order(){
```

Comment:
It's not so significant in this task, however the name of this class conflicts with the name in global object "Order".